src/lib.rs

```diff
@@ -0,0 +1,22 @@
+pub const BATCH_ID_TO_METADATA_HASH_FUNCTION_SELECTOR: [u8; 4] = [236, 203, 191, 201];
+pub const QUORUM_ADVERSARY_THRESHOLD_PERCENTAGES_FUNCTION_SELECTOR: [u8; 4] = [134, 135, 254, 174];
+pub const QUORUM_NUMBERS_REQUIRED_FUNCTION_SELECTOR: [u8; 4] = [225, 82, 52, 255];
```

Comment:
```suggestion
// Below are the function selectors for EigenDAServiceManager contract functions:
// https://github.com/Layr-Labs/eigenda/blob/4ec69dc0cb88182bb6e6fb8054e8db4d6086200d/contracts/src/core/EigenDAServiceManagerStorage.sol
pub const BATCH_ID_TO_METADATA_HASH_FUNCTION_SELECTOR: [u8; 4] = [236, 203, 191, 201];
pub const QUORUM_ADVERSARY_THRESHOLD_PERCENTAGES_FUNCTION_SELECTOR: [u8; 4] = [134, 135, 254, 174];
pub const QUORUM_NUMBERS_REQUIRED_FUNCTION_SELECTOR: [u8; 4] = [225, 82, 52, 255];
```

Reply:
Done, also added a link to `EigenDAServiceManager.sol` which is the file with the `QUORUM_ADVERSARY_THRESHOLD_PERCENTAGES` and `QUORUM_NUMBERS_REQUIRED` functions